internal/groups/service.go

```diff
@@ -217,7 +217,9 @@ func (svc service) ListGroups(ctx context.Context, token string, memberKind, mem
 	}
 
 	if len(ids) <= 0 {
-		return groups.Page{}, errors.ErrNotFound
+		return groups.Page{
```

Comment:
Why do we even consider less than 0 for IDs length (i.e. why not `if len(ids) == 0`)?

Reply:
Since len() returns `int`